.github/workflows/ci.yml

```
@@ -63,125 +62,103 @@ jobs:
-              yarn post:build
-    name: stable - ${{ matrix.settings.target }} - node@18
-    runs-on: ${{ matrix.settings.host }}
+              rustup target add aarch64-unknown-linux-musl
+              yarn build --target aarch64-unknown-linux-musl
+          - os: windows-latest
+            target: aarch64-pc-windows-msvc
+            build: yarn build --target aarch64-pc-windows-msvc
+    name: stable - ${{ matrix.target }} - Node LTS
+    runs-on: ${{ matrix.os }}
     steps:
       - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)